src/frontend/app/shared/components/list/list-types/cf-buildpacks/cf-buildpack-card/cf-buildpack-card.component.html

```diff
@@ -0,0 +1,35 @@
+<app-meta-card class="buildpack-card">
+  <app-meta-card-title class="buildpack-card__title">
+    <div class="buildpack-card__title-link">{{ row.entity.name }}</div>
+  </app-meta-card-title>
+  <app-meta-card-item>
+    <app-meta-card-key>Position</app-meta-card-key>
+    <app-meta-card-value>{{ row.entity.position }}</app-meta-card-value>
+  </app-meta-card-item>
+  <app-meta-card-item>
+    <app-meta-card-key>Enabled</app-meta-card-key>
```

Comment:
The alignment of app-meta-card-key and app-meta-card-value is out. It looks like `meta-card-item` needs
```
align-items: center;
height: 30px;
```
(the height may be situational but the align-items is needed globablly